### ISSUE TITLE
version 1.3.0 introduces a breaking change without incrementing MAJOR

### ISSUE THREAD

**Issue by binki** [js-joda/js-joda-timezone#48]
With 1.1 and 1.2, the documentation said to use this pattern to consume js-joda-timezone:

```js
const jsJoda = require('js-joda')
    .use(require('js-joda-timezone'))
```

However, starting with js-joda-timezone-1.3, that code fails. Thus, it breaks apps which consume `js-joda-timezone` using semver because `1.3.0` is compatible with `^1.1` (see tools like http://jubianchi.github.io/semver-check/):

```
ohnobinki@gibby /tmp/.private/ohnobinki/js-joda-timezone-play $ npm install -S js-joda js-joda-timezone@^1.1
npm WARN js-joda-timezone-play@1.0.0 No description
npm WARN js-joda-timezone-play@1.0.0 No repository field.

+ js-joda@1.8.2
+ js-joda-timezone@1.3.0
updated 2 packages in 0.485s
ohnobinki@gibby /tmp/.private/ohnobinki/js-joda-timezone-play $ node -e "require('js-joda').use(require('js-joda-timezone'))"
/tmp/.private/ohnobinki/js-joda-timezone-play/node_modules/js-joda/dist/js-joda.js:11366
                fn(jsJoda);
                ^

TypeError: fn is not a function
    at Object.use (/tmp/.private/ohnobinki/js-joda-timezone-play/node_modules/js-joda/dist/js-joda.js:11366:17)
    at [eval]:1:20
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at Object.runInThisContext (vm.js:139:38)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:652:30)
    at evalScript (bootstrap_node.js:463:27)
    at startup (bootstrap_node.js:164:9)
    at bootstrap_node.js:609:3
```

Please take these steps:

1. Release 1.3.1 with changes so that `require('js-joda-timezone')` returns a function instead of `undefined` (EDIT: instead of a non-invokable object).
2. If you desire to use autoPlug style, release that as `2.0.0`.

You should attempt to respect users who use semver because it reduces their maintenance burden. Additionally, `npm install -S` (or, with newer versions of `npm`, simply `npm install`) will automatically set the dependency to a caret range of the current version. That is because any release of a package matching the caret range is expected to be fully backwards compatible with prior versions. This allows the package maintainer to add backwards-compatible new features (MINOR) or fix security issues or increase platform support (PATCH) without requiring the maintainers to alter their projects.

**Comment by binki** [comment]
To make your code depending on `^1.1` future proof, you may use this pattern which supports both `1.3` and prior versions:

```js
const jsJodaTimezone = require('js-joda-timezone');
const jsJoda = require('js-joda').use(typeof jsJodaTimezone === 'function' ? jsJodaTimezone : function () {});
```

**Comment by phueper** [comment]
Hi,

thanks for reporting ... and sorry for causing this... i will try to fix this as you suggested (release 1.3.1 without the breaking change, release 2.0.0 with the new API)

**Comment by phueper** [comment]
Hi,

i just published js-joda-timezone@1.3.1 which should restore the 1.2.1 API and js-joda-timezone@2.0.0 which has the new plug/usage API... 

Thanks again for catching this and sorry for the trouble we caused.

Regards, Pattrick

P.S. I'll close this issue, feel free to comment/reopen if you have further input

**Comment by binki** [comment]
Thanks much!
